lib/pr_checker.js

```diff
@@ -152,9 +152,17 @@ class PRChecker {
     const { pr } = this;
     const { cli } = this;
     const labels = pr.labels.nodes;
-    const fast = labels.some((l) => l.name === 'code-and-learn') ||
+
+    let isFastTrack = false;
+    const fast = labels.some((l) => isFast(l.name)) ||
       (labels.length === 1 && labels[0].name === 'doc');
+
+    if (isFastTrack) {
+      cli.warn('No wait time! "fast-track" labelled PR.');
```

Comment:
Maybe just: `This PR is being fast-tracked`, they can always look up the labels in the `logIntro()` messages. Also I prefer an info over warning.